runtime/Java/src/org/antlr/v4/runtime/atn/LexerATNSimulator.java

```diff
@@ -78,8 +78,6 @@ protected void reset() {
 
 	protected final SimState prevAccept = new SimState();
 
-	public static int match_calls = 0;
```

Comment:
📝 If there is a concern about binary compatibility, we can keep this field (mark it `@Deprecated`) and just stop writing to it. However, I don't think binary compatibility for this field is a concern. I'll let @parrt make the call on it.

Reply:
Always scary but I think it's cool.